src/stylesheets/core/_utilities.scss

```diff
@@ -16,6 +16,13 @@
   left: -999em;
 }
 
+// Aria hidden helper
+@mixin aria-hidden() {
```

Comment:
@msecret I'd be careful about calling it hidden, so it doesn't get confused with the `hidden` attribute. Although that might have been just an example or it might be ok?

Also not sure I'd call the mixin `aria-hidden` either bc its not the true function of aria-hidden, just a special use case for this situation. Curious to here what are the benefits of using the mixin is too. Are we reusing this somewhere else?